common/recorder.c

```diff
@@ -229,7 +229,7 @@ static void mux_packets(struct mp_recorder_sink *rst, bool force)
         if (rst->packets[n]->keyframe)
             safe_count = n;
     }
-    if (force)
+    if (force || rst->num_packets >= QUEUE_MAX_PACKETS)
```

Comment:
Should have a short comment here to restate the commit's intention of just muxing anyway if no keyframe is seen

Reply:
Something like this?

Reply:
Yep.